### PR TITLE
Bugherd #499 - OEE

### PR DESCRIPTION
**Pull request by MWindel**
Fixed the content of the info box in OEE dashboard > Machine dashboard

Change will be cherry-picked to release 10.11., 10.10 and 10.9 if necessary.

### CODE REVIEW

File: content/oee/oee-dashboards-bundle/machine-dashboard.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Click **Andon Board** at the right of the top menu bar to switch to the **Andon
 
 Click **Edit configuration** at the right of the top menu bar to edit the machine profile in the profile configurator, see [Creating machine profiles](/oee/oee-administration/#machine-profiles). This link is only visible if the user has a role assigned with the Oeeconfigurator Amin permission enabled.
```
Comment (BeateRixen) [inline content/oee/oee-dashboards-bundle/machine-dashboard.md:20]: Amin >Admin

not part of this change but just realized :-)

Comment (MWindel) [reply content/oee/oee-dashboards-bundle/machine-dashboard.md]: Done

File: content/oee/oee-dashboards-bundle/machine-dashboard.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Click **Andon Board** at the right of the top menu bar to switch to the **Andon
...
 
->**Info:** To access the OEE Dashboards, the Oeeconfigurator type Read or Admin permissions need to be enabled for the user/role. Without the required permission you might see error messages or not see any data in the Machine dashboard.
+>**Info:** To access the OEE Dashboards, the required permission “Oeeconfigurator” with type “READ” or “ADMIN” need to be enabled for the user/role. Without the required permission you might see error messages or not see any data in the Machine dashboard.
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/oee/oee-dashboards-bundle/machine-dashboard.md:22]: Yes, I agree. Done.

**Review by mvoigt-sag — APPROVED** [review]

File: content/oee/oee-dashboards-bundle/machine-dashboard.md [hunk 3]
```diff
@@ -17,9 +17,9 @@ By default, the **Auto-refresh** option at the top left is activated and [**OEE
...
 
-Click **Edit configuration** at the right of the top menu bar to edit the machine profile in the profile configurator, see [Creating machine profiles](/oee/oee-administration/#machine-profiles). This link is only visible if the user has a role assigned with the Oeeconfigurator Amin permission enabled.
+Click **Edit configuration** at the right of the top menu bar to edit the machine profile in the profile configurator, see [Creating machine profiles](/oee/oee-administration/#machine-profiles). This link is only visible if the user has a role assigned with the Oeeconfigurator Admin permission enabled.
 
->**Info:** To access the OEE Dashboards, the Oeeconfigurator type Read or Admin permissions need to be enabled for the user/role. Without the required permission you might see error messages or not see any data in the Machine dashboard.
+>**Info:** To access the OEE dashboards, the permission “Oeeconfigurator” with type READ or ADMIN must be enabled for the user/role. Without the required permission you might see error messages or not see any data in the machine dashboard.
```
Comment (rssag) [inline content/oee/oee-dashboards-bundle/machine-dashboard.md:22]:
```suggestion
>**Info:** To access the OEE dashboards, the permission "Oeeconfigurator" with type READ or ADMIN must be enabled for the user/role. Without the required permission you might see error messages or not see any data in the machine dashboard.
```

**Review by BeateRixen — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Review by mvoigt-sag — APPROVED** [review]
